bin/npm.ps1

```diff
@@ -1,35 +1,28 @@
 #!/usr/bin/env pwsh
-$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent
 
-$exe=""
-if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
-  # Fix case when both the Windows and Linux builds of Node
-  # are installed in the same directory
-  $exe=".exe"
+$NODE_EXE="$PSScriptRoot/node.exe"
+if (-not (Test-Path $NODE_EXE)) {
+  $NODE_EXE="$PSScriptRoot/node"
```

Comment:
`npm.cmd` doesn't do this. It just checks for `node.exe` in `%~dp0` and then falls back to `node` with no path.  Is there a reason we do this interim step?

Reply:
The previous version of `bin/npm.ps1` tried to make affordances for both "node" and "node.exe" existing, which is from https://github.com/npm/cli/pull/6548/files

`bin/npm` also makes the same check, so I opted for that here as well. I think it's more likely that the `cmd` scripts should be updated to do the same.

Reply:
Good idea